package-lock.json

```diff
@@ -9600,6 +9600,27 @@
         "node": ">=16.3.0"
       }
     },
+    "node_modules/puppeteer-core/node_modules/ws": {
```

Comment:
Interesting this gets hoisted...I tested locally and didn't see any puppeteer related issues. Commenting on this for posterity (just in case)